tests/testthat/test-key_colnames.R

```diff
@@ -35,12 +15,12 @@ test_that("epi_keys_mold extracts time_value and geo_value, but not raw", {
     fit(data = case_death_rate_subset)
 
   expect_setequal(
-    epi_keys_mold(my_workflow$pre$mold),
+    key_colnames(my_workflow$pre$mold),
     c("time_value", "geo_value")
   )
```

Comment:
question: will key_colnames also work on `my_workflow`? if so, let's add a test?

fix (minor): test name is out of date.